lib/ansible/modules/monitoring/zabbix/zabbix_user_group.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, OVH SAS
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: zabbix_user_group
+short_description: Create/update/delete/dump Zabbix user group
+description:
+    - Create/update/delete/dump Zabbix user group.
+version_added: "2.9"
+author:
+    - Emmanuel Riviere (@emriver)
+requirements:
+    - "python >= 2.7"
+    - "zabbix-api >= 0.5.3"
+options:
+    name:
+        description:
+            - Name of the Zabbix user group
+        required: true
+        type: str
+    debug_mode:
+        description:
+            - Enable or disable debug mode.
+        required: false
+        type: bool
+        default: no
+    gui_access:
+        description:
+            - Frontend authentication method.
+        required: false
+        type: str
+        choices: [default,internal,ldap,disabled]
+        default: "default"
+    users_status:
+        description:
+            - Whether the group is enabled of disabled
+        required: false
+        type: str
+        choices: [enabled, disabled]
+        default: "enabled"
+    rights:
+        description:
+            - List of permissions associated to the group
+            - Each right must contain a host_group and a permission (denied,RO,RW)
```

Comment:
I would prefer the term `permissions` for this option as it is called so in both Zabbix front end and API. On the other hand `permissions['permission']`  can be misleading.

Reply:
I may have misunderstood but it seems to be called `rights` at least in the API  (https://www.zabbix.com/documentation/4.0/manual/api/reference/usergroup/create)

    Parameter | Type | Description
    rights | object/array | Permissions to assign to the group

Reply:
Yes, I think you're right. I just looked at [this](https://www.zabbix.com/documentation/4.0/manual/api/reference/usergroup/object#permission) section.

Reply:
@rubentsirunyan thanks a lot for taking a look at this! :)